deployment/terraform/storage.tf

```diff
@@ -32,4 +32,13 @@ resource "aws_s3_bucket" "storage" {
     Environment = "${var.environment}"
     Project     = "${var.project}"
   }
+
+  lifecycle_rule {
```

Comment:
This resource should be parameterized with the `environment`. I'm not sure what terraform will do when we do a prod deploy and it detects that this resource exists, but the production state never modified it. Also, it's better to keep our stack resources separate, such that staging tests don't interrupt any ongoing prod analyses.

Otherwise this config appears correct. I ran a staging deploy to the point of `./scripts/infra plan` and got:
```
[Standard task definition changes omitted]

~ aws_s3_bucket.storage
    lifecycle_rule.#:                                                    "0" => "1"
    lifecycle_rule.0.enabled:                                            "" => "true"
    lifecycle_rule.0.expiration.#:                                       "0" => "1"
    lifecycle_rule.0.expiration.3023609085.date:                         "" => ""
    lifecycle_rule.0.expiration.3023609085.days:                         "" => "7"
    lifecycle_rule.0.expiration.3023609085.expired_object_delete_marker: "" => ""
    lifecycle_rule.0.id:                                                 "" => "osm_extracts"
    lifecycle_rule.0.prefix:                                             "" => "/osm-data-cache"


Plan: 3 to add, 3 to change, 3 to destroy.
```

Reply:
Oops, disregard, misunderstood that this was a global rule and not already scoped within the storage bucket.